### ISSUE TITLE
Error in code?

### ISSUE THREAD

**Issue by Ownwn** [csjh/SkyblockSniper#2]
After running the file for about 30 seconds, I get this error:
![image](https://user-images.githubusercontent.com/64725743/132824076-14f6828c-efc6-4fb7-b6bb-611946956580.png)

**Comment by csjh** [comment]
Yes, known issue, I don't know what causes it, I suspect it's something network related but I can't test it because it doesn't happen to me.

**Comment by BattleB0t** [comment]
![image](https://user-images.githubusercontent.com/76824591/132924066-4df94fe1-dfbf-46d3-89d1-b558bf9bdd6a.png)
This also happens to me

**Comment by Vikame** [comment]
I want to preface this by saying I have very little knowledge in Python as a whole, but I did a bit of research on this and it seems to be caused by the main thread dying.

You made a [change](https://github.com/csjh/SkyblockSniper/commit/989812227b287c3fd924e09175485e085deef842) previously that likely would have fixed this issue, but [due to an issue with the python recursion limit](https://github.com/csjh/SkyblockSniper/commit/c3dc7efe1e2bb9ad05cb9562c490c6531162e687) you decided to revert back to the use of `threading.Timer()`

Due to `threading.Timer()` [spawning a new thread](https://docs.python.org/3/library/threading.html#timer-objects) every time it is called, this leaves your main thread doing nothing after the first execution of `main()` and `dostuff()`.

If I was any good with Python I would submit a pull request with fixes for the issue, but I hope everything I detailed above will be sufficient information to come up with a fix.

**Comment by actiniumbeasty** [comment]
This happe
![image](https://user-images.githubusercontent.com/73411083/133179849-a3493df0-e627-4cba-aff8-be82b2b0b219.png)
ns to me as well

**Comment by Ownwn** [comment]
Oddly enough, it doesn't happen on my PC but it does on my laptop. Maybe it's something to do with Python using processing power?

**Comment by actiniumbeasty** [comment]
Hmmmm, that could be it.

**Comment by Antenaks** [comment]
![pip](https://user-images.githubusercontent.com/92647153/137598570-f726eff6-ebfb-415d-ad1d-03d8f3392775.png)
pls help

**Comment by actiniumbeasty** [comment]
if you use windows 10/11, you can go to the microsoft store and install the
latest version of python, that worked for me. If that doesn't work i can't
help you
